src/middlewares/AuthorizationMiddleware.test.ts

```diff
@@ -0,0 +1,43 @@
+import sinon from "sinon";
+import getRequestMock from "../mocks/RequestMock";
+import AdminAuthorization from "./AuthorizationMiddleware";
```

Comment:
We should not name this import as `AdminAuthorization` as it is going to be a base class for us. Instead import it as class name and add test cases for admin users.